README.md

```diff
@@ -46,6 +46,30 @@ Telescope is **optional**.
 }
 ```
 
+<details>
+<summary>If you get a <code>no parser for 'yaml' language</code> error message when calling <code>:YAMLTelescope</code></summary>
+	
+This means that you need to install a parser such as [<code>tree-sitter-yaml</code>](https://github.com/ikatyang/tree-sitter-yaml).
+
+Then you need to enable it in your nvim config. Here is an example.
+
+```lua
+{
+  "nvim-treesitter/nvim-treesitter",
+  build = ":TSUpdate",
+  config = function()
+    require("nvim-treesitter.configs").setup({
+      ensure_installed = { "yaml" },
+      highlight = {
+        enable = true,
+        disable = {},
+      },
+  })
+  end,
+}
+```
```

Comment:
I don't think we should be teaching people how to use `nvim-treesitter/nvim-treesitter` — not because I don't want to be nice to people, but because this adds maintenance labor:
* If `nvim-treesitter/nvim-treesitter` changes their API, these docs might get misleading.
* If the reader used a different plugin manager, these docs get misleading.
* If we decide to add instructions for many plugin managers, these docs gets even more difficult to be maintained tracking down every API change out there.

Usually, I would just point users to `nvim-treesitter/nvim-treesitter` docs instead. What do you think? And if you are strong about documenting `nvim-treesitter/nvim-treesitter` here, let's keep it minimal:

```suggestion
Then you need to enable it in your nvim config. Here is an example.

```lua
{
  "nvim-treesitter/nvim-treesitter",
  build = ":TSUpdate",
  config = function()
    require("nvim-treesitter.configs").setup({
      ensure_installed = { "yaml" },
      },
  })
  end,
}
```

```